Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1375,11 +1403,32 @@ def run_polling_command(client, args: dict, cmd: str, upload_function: Callable,
             timeout_in_seconds=6000)
 
         command_result = CommandResults(scheduled_command=scheduled_command)
+    elif post_function:
+        # Validate the polling results
+        validate_submit_file_polling_results(command_result)
     return command_result
 
+def validate_submit_file_polling_results(command_results: list[CommandResults] | CommandResults):
+    """Validate the results of the polling function when submitting a file for analysis.
+
+    Args:
+        command_results (list[CommandResults] | CommandResults): The results of the polling method.
+
+    Raises:
+        DemistoException: If the results contain an error message, stating that the sandbox analysis was not able to run
+        properly.
+    """
+    command_results = command_results if isinstance(command_results, list) else [command_results]
+    for command_result in command_results:
```

Comment:
```suggestion
    for command_result in always_iterable(command_results, CommandResults):
```
make sure to import `always_iterable` from `more_itertools` _(included in all our py3 images)_

Reply:
Nice